src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/RuntimeHelpers.cs

```diff
@@ -65,6 +65,26 @@ public static object GetUninitializedObject(Type type)
             return GetUninitializedObjectInternal(type);
         }
 
+        public static void ExecuteCodeWithGuaranteedCleanup(TryCode code, CleanupCode backoutCode, object userData)
```

Comment:
nit: TryCode and CleanupCode delegates uses `object?` so it should match to userData

Reply:
Good catch. The old signature had a ?... looks like it just got lost in the move.